src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitFormIncident/index.js

```diff
@@ -27,10 +27,18 @@ const IncidentSplitFormIncident = ({ parentIncident, subcategories, register })
     []
   );
 
+  const incidentSplitFormIncidentRefs = useMemo(() => Array(splitCount).fill().map(() => createRef()), [splitCount]);
```

Comment:
Prefixing the refs with `incidentSplitFormIncident` inside the `IncidentSplitFormIncident` is redundant.